.travis.yml

```diff
@@ -11,8 +11,11 @@ python:
 cache:
   pip: true
 
+# TODO: if deployment to PyPi is desired, change 'DEPLOY_PYPI' to "true",
+#       or remove the env block entirely and remove the condition in the
+#       deploy block.
 env:
-  - DEPLOY_PYPI="true"
+  - DEPLOY_PYPI="false"
```

Comment:
Once the hash is pasted into .travis.yml, please change this to `"true"`.

Reply:
added and changed.  hopefully i got it right.